gcp/biganimal_GCP_baserole.yaml

```diff
@@ -0,0 +1,542 @@
+- dns.managedZones.create
+- dns.managedZones.delete
+- dns.managedZones.get
+- dns.managedZones.getIamPolicy
+- dns.managedZones.list
+- dns.managedZones.update
+- dns.networks.bindDNSResponsePolicy
+- dns.networks.bindPrivateDNSPolicy
+- dns.networks.bindPrivateDNSZone
+- dns.networks.targetWithPeeringZone
+- dns.networks.useHealthSignals
+- dns.projects.get
+- dns.resourceRecordSets.create
+- dns.resourceRecordSets.delete
+- dns.resourceRecordSets.get
+- dns.resourceRecordSets.list
+- dns.resourceRecordSets.update
+- iam.roles.create
+- iam.roles.delete
+- iam.roles.get
+- iam.roles.list
+- iam.roles.update
+- iam.serviceAccounts.actAs
+- iam.serviceAccounts.create
+- iam.serviceAccounts.delete
+- iam.serviceAccounts.disable
+- iam.serviceAccounts.enable
+- iam.serviceAccounts.get
+- iam.serviceAccounts.getIamPolicy
+- iam.serviceAccounts.list
+- iam.serviceAccounts.setIamPolicy
+- iam.serviceAccounts.undelete
+- iam.serviceAccounts.update
+- resourcemanager.projects.get
+- resourcemanager.projects.getIamPolicy
+- resourcemanager.projects.setIamPolicy
+- run.configurations.get
+- run.configurations.list
+- run.executions.delete
+- run.executions.get
+- run.executions.list
+- run.jobs.create
+- run.jobs.delete
+- run.jobs.get
+- run.jobs.getIamPolicy
+- run.jobs.list
+- run.jobs.run
+- run.jobs.runWithOverrides
+- run.jobs.setIamPolicy
+- run.jobs.update
+- run.locations.list
+- run.operations.delete
+- run.operations.get
+- run.operations.list
+- secretmanager.locations.get
+- secretmanager.locations.list
+- secretmanager.secrets.create
+- secretmanager.secrets.delete
+- secretmanager.secrets.get
+- secretmanager.secrets.getIamPolicy
+- secretmanager.secrets.list
+- secretmanager.secrets.setIamPolicy
+- secretmanager.secrets.update
+- secretmanager.versions.access
+- secretmanager.versions.add
+- secretmanager.versions.destroy
+- secretmanager.versions.disable
+- secretmanager.versions.enable
+- secretmanager.versions.get
+- secretmanager.versions.list
+- servicedirectory.namespaces.create
+- servicedirectory.namespaces.delete
+- servicedirectory.services.create
+- servicedirectory.services.delete
+- serviceusage.quotas.get
+- serviceusage.quotas.update
+- serviceusage.services.get
+- serviceusage.services.list
+- storage.buckets.create
+- storage.buckets.createTagBinding
+- storage.buckets.delete
+- storage.buckets.deleteTagBinding
+- storage.buckets.get
+- storage.buckets.getIamPolicy
+- storage.buckets.list
+- storage.buckets.listEffectiveTags
+- storage.buckets.listTagBindings
+- storage.buckets.setIamPolicy
+- storage.buckets.update
+- storage.objects.create
+- storage.objects.delete
+- storage.objects.get
+- storage.objects.getIamPolicy
+- storage.objects.list
+- storage.objects.setIamPolicy
+- storage.objects.update
+- vpcaccess.connectors.create
+- vpcaccess.connectors.delete
+- vpcaccess.connectors.get
+- vpcaccess.connectors.list
+- vpcaccess.connectors.update
+- vpcaccess.operations.get
+- vpcaccess.operations.list
```

Comment:
duplicate?

Reply:
nope, last 2 are about `vpcaccess.operations` that I forgot previously (this is the reason of my last PR update), the previous one are about `vpcaccess.connectors`

Reply:
Right, speedreading ftl. '^^